src/models/accounts.js

```diff
@@ -49,20 +48,25 @@ const accounts = new Schema({
         max: 3,
         require: true
     },
+    codeConfirmation: {
+        type: Number,
+        require: true
+    },
     dateCreation: { 
-        type: Date, 
-        default: Date.now 
+        type: String,
+        require: true
     },
-    idCity: [{ type: Schema.Types.ObjectId, ref: 'Cities' }]
+    idCity: [{ type: Schema.Types.ObjectId, ref: 'Cities'}],
+    discussions: [{ type: Schema.Types.ObjectId, ref: 'Discussions'}]
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    discussions: [{ type: Schema.Types.ObjectId, ref: "Discussions"}]
```